<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ao_waiter</class>
 <widget class="QMainWindow" name="ao_waiter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавление заказов</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../static/resource.qrc">
    <normaloff>:/img/img/Logotip.png</normaloff>:/img/img/Logotip.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>691</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="le_table">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>381</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid rgb(153, 102, 51);
background-color: rgb(255, 255, 255);
border-radius: 20px;</string>
     </property>
     <property name="placeholderText">
      <string>Введите столик</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_c_clients">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>130</y>
       <width>381</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid rgb(153, 102, 51);
background-color: rgb(255, 255, 255);
border-radius: 20px;</string>
     </property>
     <property name="placeholderText">
      <string>Введите количество клиентов</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_add">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>381</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid rgb(153, 102, 51);
background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
     </property>
     <property name="currentText">
      <string/>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="dishes_big">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>230</y>
       <width>381</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid rgb(153, 102, 51);
background-color: rgb(255, 255, 255);
border-radius: 20px;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_status">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>490</y>
       <width>381</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid rgb(153, 102, 51);
background-color: rgb(255, 255, 255);
border-radius: 20px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Введите статус</string>
     </property>
    </widget>
    <widget class="QLabel" name="l_dishes">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>200</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Введите блюда</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="drinks_big">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>370</y>
       <width>381</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid rgb(153, 102, 51);
background-color: rgb(255, 255, 255);
border-radius: 20px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="l_dishes_3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>340</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Введите напитки</string>
     </property>
    </widget>
    <widget class="QPushButton" name="back_on_order">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>610</y>
       <width>381</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: rgba(153, 102, 51);
    border: 2px solid rgba(100, 50, 51);
    border-radius: 20px;
    color: white;
    font-weight: bold;
    font-size: 11pt;
}
QPushButton:hover {
    background-color: rgba(73, 140, 81);
}
QPushButton:pressed {
    background-color: rgba(73, 140, 81);
}
</string>
     </property>
     <property name="text">
      <string>Назад к главной панели &quot;Заказы&quot;</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_save">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>550</y>
       <width>381</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: rgba(153, 102, 51);
    border: 2px solid rgba(100, 50, 51);
    border-radius: 20px;
    color: white;
    font-weight: bold;
    font-size: 12pt;
}
QPushButton:hover {
    background-color: rgba(73, 140, 81);
}
QPushButton:pressed {
    background-color: rgba(73, 140, 81);
}
</string>
     </property>
     <property name="text">
      <string>Сохранить</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../../static/resource.qrc"/>
 </resources>
 <connections/>
</ui>
